ppx-lib/gen_js_api_ppx.ml

```diff
@@ -1219,7 +1227,15 @@ and gen_funs ~global_attrs p =
     match p.ptype_manifest, p.ptype_kind with
     | Some ty, Ptype_abstract ->
         let ty = parse_typ ~global_attrs ty in
-        lazy (js2ml_fun ty), lazy (ml2js_fun ty)
+        (match p.ptype_params, ty with
+        | [], _ ->
+          lazy (js2ml_fun ty), lazy (ml2js_fun ty)
+        | [{ptyp_desc = Ptyp_var _; ptyp_loc = _; ptyp_attributes = _; ptyp_loc_stack = _}, Invariant], Js ->
+          let s = "_" in  
```

Comment:
This covers the case with abstract types with type variable, e.g.

```ocaml
    type 'value t = Ojs.t
```
will generate:
```
    type 'value t = Ojs.t
    let rec (t_of_js : (Ojs.t -> 'value) -> Ojs.t -> 'value t) =
      fun _ -> fun x171 -> x171
    and (t_to_js : ('value -> Ojs.t) -> 'value t -> Ojs.t) =
      fun _ -> fun x170 -> x170
```